README.md

```diff
@@ -0,0 +1,14 @@
+# LAP3 Coding Challenge
+This is our lap 3 coding challenge which covers using a Javascript framework called React to build a web app which extracts github repo information for our users.
+
+# Installion and Usage
+- Clone or downlaod the repo.
+- run npm i to install the packages.
+- run npm run dev to start the app.
+
+# Wins and Challenges
+## Wins
+- We were able to build a fully functional web app.
+
+## Challenges
+- Encountered issues with rendering the information on the page.
```

Comment:
what kind of issues? Did you manage to overcome those issues?

Reply:
First of all sorry for the github mess, thanks for taking the time to go through the code anyway. Our main issue was when rendering all the data it would automatically call the API when the state first loaded, resulting in trying to map something empty. This took a while to diagnose, as we forgot to initialise our state with [] and also didn't at first do any checks to see if the state was loading/username was passed to it before we ran it.